src/compile/compile.ts

```diff
@@ -70,7 +70,8 @@ export function assembleRootGroup(model: Model) {
             width: {field: 'width'},
             height: {field: 'height'}
           },
-          model.assembleParentGroupProperties(model.config().cell)
+          // what is this?
```

Comment:
Adding cell config to the parent group?  
But I agree with you.  This looks doubtful to be here.   
Maybe file an issue and let me handle in a separate PR?